lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserClang.cpp

```diff
@@ -2345,11 +2345,6 @@ bool DWARFASTParserClang::CompleteTypeFromDWARF(const DWARFDIE &die,
 
   if (!die)
     return false;
-  ParsedDWARFTypeAttributes attrs(die);
```

Comment:
In case anyone's wondering, Putting this check back in would prevent the crash from 7fdbc30b445286f03203e16d0be067c25c6f0df0, but it would not actually make the code correct -- lldb would now just say the class has no definition instead of crashing.